builder/dockerfile/dispatchers.go

```diff
@@ -71,6 +71,10 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 			return errBlankCommandNames("ENV")
 		}
 
+		if runtime.GOOS == "windows" {
+			// Case insensitive environment variables on Windows
+			args[j] = strings.ToUpper(args[j])
```

Comment:
Why do we need to force a case in commit? Isn't case insensitive comparison enough?